### PR TITLE
Now handles multiple commands

### PR DESCRIPTION
**Pull request by dibyadas**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -16,48 +16,76 @@
...
+		 Receives commands directed at the bot and determines if they
+		 are valid commands. If so, then acts on the commands. If not,
+		 returns back what it needs for clarification.
+	 """
+	keys = []
+	response = ""
+	msg = (msg.replace("@bhattu","")).strip()
+	comp = msg.split()
+	seniors_msg = ""
+	freshers_msg = ""
```
Comment (americast) [inline main.py:29]: @dibyadas Don't do this ^ ...
In this case, the code is restricted to only freshers and seniors...we should be able to add more commands by simply modifying the JSON, not the code...

**Comment by americast** [comment]
@dibyadas Don't restrict the usability of the code to only "freshers" and "seniors"...we should be able to add more commands by simply modifying the JSON file, not the code...

File: main.py (hunk 1 above)
Comment (dibyadas) [reply main.py:29]: Yeah. That's what. I realized it after making the PR. This has a few more errors too. Just ignore this PR for now. I will correct and update the PR within a few days. Day after tomorrow  most probably.

**Comment by americast** [comment]
@dibyadas You may close the PR now and reopen it later...

**Pull request closed without merging**